security/i2pd/Portfile

```diff
@@ -0,0 +1,164 @@
+            copy ${prefix}/etc/${name}/${f} ${destroot}${prefix}/etc/${name}/${f}.mp_backup
+            file attributes ${destroot}${prefix}/etc/${name}/${f}.mp_backup -owner ${i2puser} -group ${i2puser} -permissions 0664
+        }
+    }
+}
+
+post-install {
+    ui_msg "******************************************************"
+    ui_msg "*"
+    ui_msg "* i2pd status can be checked on: https://127.0.0.1:7070"
+    ui_msg "* You can edit i2pd settings in ${i2pddir}"
+    ui_msg "* See also: https://i2pd.readthedocs.io/en/latest/user-guide/configuration"
+    ui_msg "* Re using i2pd with web-browser: https://github.com/PurpleI2P/i2pdbrowser"
+    ui_msg "*"
+    ui_msg "******************************************************"
+}
+
+post-activate {
+    # DataDirectory and PID file Ddirectory permissions
+    system "chown ${i2puser}:${i2puser} ${i2pddir}"
+    system "chmod 0755 ${i2pddir}"
+    system "chown ${i2puser}:${i2puser} ${prefix}/var/run/${name}"
+    system "chmod 0755 ${prefix}/var/run/${name}"
+
+    # restore conf files
+    foreach f {i2pd.conf subscriptions.txt tunnels.conf tunnels.d} {
+        if {![file exists ${prefix}/etc/${name}/${f}]} {
+            if {[file exists ${prefix}/etc/${name}/${f}.mp_backup]} {
+                copy ${prefix}/etc/${name}/${f}.mp_backup ${prefix}/etc/${name}/${f}
+            } else {
+                copy ${prefix}/etc/${name}/${f}.new ${prefix}/etc/${name}/${f}
+            }
+            file attributes ${prefix}/etc/${name}/${f} -owner ${i2puser} -group ${i2puser} -permissions 0664
+        }
+    }
+
+    if {![file exists ${i2pddir}]} {
+        xinstall -m 0664 -d ${i2pddir}
+    }
+    ln -sf ${prefix}/share/${name}/certificates ${i2pddir}/certificates
+    ln -sf ${prefix}/etc/${name}/tunnels.d ${i2pddir}/tunnels.d
+    ln -sf ${prefix}/etc/${name}/i2pd.conf ${i2pddir}/i2pd.conf
+    ln -sf ${prefix}/etc/${name}/subscriptions.txt ${i2pddir}/subscriptions.txt
+    ln -sf ${prefix}/etc/${name}/tunnels.conf ${i2pddir}/tunnels.conf
+}
+
+startupitem.create      yes
+# Option --service has no effect on macOS: https://i2pd.readthedocs.io/en/latest/user-guide/configuration
+startupitem.executable  ${prefix}/bin/i2pd --datadir=${i2pddir} --conf=${i2pddir}/i2pd.conf --certsdir=${i2pddir}/certificates --tunconf=${i2pddir}/tunnels.conf --log=${prefix}/var/log/${name} --daemon
+
+destroot.keepdirs       ${destroot}${i2pddir} \
+                        ${destroot}${prefix}/var/run/${name} \
+                        ${destroot}${prefix}/var/log/${name}
+
+variant upnp description "Support for UPNP" {
+    depends_lib-append  port:miniupnpc
+    configure.args-append \
+                        -DWITH_UPNP=ON
+    configure.ldflags-append \
+                        "-ldl -lminiupnpc"
+}
+
+default_variants        +upnp
+
+depends_test-append     port:gmake
+
+test.run                yes
+test.target             all
```

Comment:
are you sure that target `all` is for test? I've expected here `check` or something like that.

Reply:
Gonna check that. Could be the case that I overlooked it.

Reply:
I've missed that you changed `test.dir`.

Reply:
> I've missed that you changed `test.dir`.

Let me check if that actually works as supposed though :)